RockWeb/Blocks/CheckIn/CheckinGroupList.ascx.cs

```diff
@@ -298,7 +298,7 @@ private string BuildHeirarchy( List<int> groupTypeIds, int? campusId )
                     {
                         if ( groupType.ChildGroupTypes.Count > 0 )
                         {
-                            BuildHeirarchy( groupType.ChildGroupTypes.Select( t => t.Id ).ToList(), campusId );
+                            content.Append( BuildHeirarchy( groupType.ChildGroupTypes.Select( t => t.Id ).ToList(), campusId ) );
```

Comment:
@cabal95 any chance you can fix the spelling of this method name while you're at it? 😆 

BuildHeirarchy -> BuildHierarchy